pkgs/servers/gotify/ui.nix

```diff
@@ -10,16 +10,52 @@ yarn2nix-moretea.mkYarnPackage rec {
 
   version = import ./version.nix;
 
-  src_all = fetchFromGitHub {
-    owner = "gotify";
-    repo = "server";
-    rev = "v${version}";
-    sha256 = import ./source-sha.nix;
+  src_all = applyPatches {
+    src = fetchFromGitHub {
+      owner = "gotify";
+      repo = "server";
+      rev = "v${version}";
+      sha256 = import ./source-sha.nix;
+    };
+    postPatch = ''
+      substituteInPlace ui/yarn.lock \
+        --replace \
+          "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001237.tgz" \
+          "https___registry.npmjs.org_caniuse_lite___caniuse_lite_1.0.30001237.tgz"
```

Comment:
I'm not sure I understood what's the cause for this issue, but perhaps it should be reported to `yarn2nix` project or to `yarn`?

Reply:
I think that yarn wrongly recognizes it as some kind of external dependency, so we override it with the `___`-style that's used for the artificial offline cache.

However I don't think that yarn really cares about people trying to re-implement this somehow.